packages/image/README.md

```diff
@@ -0,0 +1,126 @@
+# `@sveltejs/image`
+
+**WARNING**: This package is experimental. It uses pre-1.0 versioning and may introduce breaking changes with every minor version release.
+
+This package aims to bring a plug and play image component to SvelteKit that is opinionated enough so you don't have to worry about the details, yet flexible enough for more advanced use cases or tweaks. It serves a smaller file format like `avif` or `webp` and uses the `srcset` and `sizes` attributes of the `img` tag to provide resized images suitable for various device sizes, which for example results in smaller images downloaded for mobile.
+
+## Setup
+
+Install:
+
+```bash
+npm install --save @sveltejs/image
+```
+
+Adjust `vite.config.js`:
+
+```diff
++import { images } from '@sveltejs/image/vite';
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
++		images({ providers: { default: '@sveltejs/image/providers/<choose one>' } }),
```

Comment:
perhaps we could remove the `/providers/` portion as it's getting a bit long and things like `@sveltejs/image/vercel` or `@sveltejs/image/netlify` is pretty explanatory and won't conflict with anything else
```suggestion
+		images({ providers: { default: '@sveltejs/image/<choose one>' } }),
```

Reply:
i like the explicit naming more. prevents conflicts too if future possible providers collided with a preexisting name.

How do you plan to have 3rd party providers added? PR here or is it possible to publish them separately eg `@cool/svelte-image-provider`

Reply:
Also vote for keeping the longer import. Any string that's a valid import is allowed there, you could even do a relative import (though we need to add support for that).